lib/fastlane/plugin/wpmreleasetoolkit/actions/configure/configure_apply_action.rb

```diff
@@ -77,10 +77,10 @@ def self.apply_file(file_reference, force)
           date_string = Time.now.strftime('%m-%d-%Y--%H-%M-%S')
 
           backup_path = base
-                        .concat('-')            # Handy-dandy separator
-                        .concat(date_string)    # date string to allow multiple backups
-                        .concat(extension)      # and the original file extension
-                        .concat('.bak')        # add the .bak file extension - easier to .gitignore
+                        .concat('-') # Handy-dandy separator
```

Comment:
I'm not a huge fan of how it doesn't allow aligned comments like this, but it's worth the trade-off everywhere else 😅

Reply:
Yeah I saw that too. Thought about adding a `# rubocop:disable` inline exception for those lines but figured it was not worth it after all.